CMakeLists.txt

```diff
@@ -458,7 +458,7 @@ if(ENABLE_MASON)
 
 else()
 
-  find_package(Boost 1.49.0 REQUIRED COMPONENTS ${BOOST_COMPONENTS})
+  find_package(Boost 1.54 REQUIRED COMPONENTS ${BOOST_COMPONENTS})
```

Comment:
I can't test this (I don't have a Windows setup), but feel free to add conditional checks to the `CMakeLists.txt` for Windows. If it's too hard or ugly to do, documenting this in the Wiki is good enough :+1: